### DISCUSSION TITLE
Google Summer of Code 2022

### DISCUSSION THREAD

**Discussion by ssheorey** [isl-org/Open3D#4786] [GSoC]
Hello everyone,

Open3D has applied for Google Summer of Code for 2022. Check out our information and ideas page:

https://github.com/isl-org/Open3D/wiki/Google-Summer-of-Code-2022

Here is the Open3D page on GSoC's website:

https://summerofcode.withgoogle.com/programs/2022/organizations/open3d-team

If you have any questions, go ahead and ask them here.

--
Open3D team.

**Comment by midopooler** [comment]
Does it have any relevance to Unity3D development?

**Reply by ssheorey** [reply]
No.

**Comment by ZhengyuDiao** [comment]
What is the email address of admins, they don't seem to be publicly available

**Reply by ssheorey** [reply]
At the bottom of the ideas list:
[Open3D GSoC Admins](https://github.com/isl-org/Open3D/wiki/Google-Summer-of-Code-2022#open3d-gsoc-admins)

    Yixing Lao (administrator AT ivcl DOT org)
    Sameer Sheorey (sameer DOT sheorey AT ivcl DOT org)

**Reply by ZhengyuDiao** [reply]
Thanks.

**Comment by Jia1018** [comment]
What should I do if I'd love to work over the idea "Open3D-ML 3D Object detection model" as a part of GSOC 2022?

**Reply by ssheorey** [reply]
Reach out to the corresponding mentors for your interested idea, for example:
@sanskar107 @benjaminum can you help @Jia1018 ?

**Reply by ssheorey** [reply]
If you are ready to apply, please send your resume to the GSoC admins. (see instructions in the wiki).

**Reply by Jia1018** [reply]
OK, thanks a lot.

**Comment by deepanshi-s** [comment]
I had mailed Sameer (sameer.sheorey@ivcl.org) and Yixing (administrator@ivcl.org) last week but haven't heard back. Do I need to mail the mentors of the project directly?

**Reply by ssheorey** [reply]
Thanks @deepanshi-s, we'll get back to you shortly.

**Comment by prikmm** [comment]
I had emailed @ssheorey or @yxlao showing interest "Open3D-ML 3D Object detection model" and requesting a starter task for the same but have not yet heard back from them. Can I please get an update on this matter?

**Reply by ssheorey** [reply]
Hi @prikmm thanks for sending over your CV. We will get back to you shortly.

**Comment by Jia1018** [comment]
Since I have seen a lot of students interested in "Open3D-ML 3D Object detection model" , can I please know how many people are finally needed for the project?

**Reply by ssheorey** [reply]
Hi @Jia1018 we are looking for one person for each project idea.

**Comment by JyothsnaMS** [comment]
Hello! Is there any slack or discord server I can join for GSoC 2022 regarding any communications related to projects?

**Reply by ssheorey** [reply]
Please see here:

https://github.com/isl-org/Open3D/wiki/Google-Summer-of-Code-2022#forum

**Comment by ssheorey** [comment]
Hi all, we have sent out qualification assignments to everyone who sent us their CV. If we missed anyone, or have not reached out to us yet, please email us (administrator@ivcl.org and sameer.sheorey@ivcl.org) with your CV and project idea of interest. Please submit your assignments by Friday April 1, 5pm UTC (use Google to convert to your local time).

**Reply by MRUNIQUEGUPTA** [reply]
hi all, how we can know more about the ideas and project listed in gsoc any one please help me .

**Comment by ELLIPSIS009** [comment]
I had emailed @ssheorey  or @yxlao showing interest "Open3D visualization on cloud notebooks" but have not heard back from them. Can you please tell me if the applications are still open or have I missed any step along the way? Thank You

**Comment by shaishav06** [comment]
Who is mentor of this **Open3D-ML 3D Object detection model** Project? 
Please provide me best guide.

**Comment by GregoryLi0** [comment]
I have fully accomplishd the assignment before April 10, and did a lot of research. But I haven't received an email about the assignment result yet or any other email from @ssheorey or @yxlao. I'm in a hurry because of the deadline.... :(

**Comment by kqdidit119** [comment]
Is this Organization participating in GSoC 2023? Will it be there for projects?
